src/main/java/au/gov/nla/dlir/models/correction/CorrectionStats.java

```diff
@@ -0,0 +1,18 @@
+package au.gov.nla.dlir.models.correction;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class CorrectionStats {
+
+  private String yyyymm;
```

Comment:
I assume this is adhering to an existing spec or something, cos all these fieldnames are weird.

Reply:
🤣 That's what the columns are named in the database.

Reply:
Yep, figures.